autogpt_platform/backend/backend/copilot/graphiti/client.py

```
@@ -22,14 +22,18 @@
 # "got Future attached to a different loop". Scope the cache (and its lock)
 # per running loop so each loop gets its own clients.
 class _LoopState:
-    __slots__ = ("cache", "lock")
+    __slots__ = ("cache", "lock", "indexed")
```

Ruff (0.16.1)

[warning] `_LoopState.__slots__` is not sorted

Apply a natural sort to `_LoopState.__slots__`

(RUF023)